app/views/payments/_payment.html.erb

```diff
@@ -0,0 +1,12 @@
+<div id="<%= dom_id payment %>">
+  <p>
+    <strong>Name:</strong>
+    <%= payment.name %>
+  </p>
+
+  <p>
+    <strong>Amount:</strong>
+    <%= payment.amount %>
+  </p>
```

Comment:
- [ ] I would recommend to keep the color same as the given design. Also, include the date when it is created as given in the design just below the payment name. Please update it.

### Your design
![Screenshot from 2022-04-22 01-30-00](https://user-images.githubusercontent.com/26767383/164544057-578f5034-5865-4528-bf00-cff872e80565.png)


### Given design
![Screenshot from 2022-04-22 01-26-06](https://user-images.githubusercontent.com/26767383/164544077-24af9d25-28bf-4b05-bc71-70d09a955452.png)